src/resources/User/User.ts

```diff
@@ -47,4 +47,12 @@ export default class User {
       },
     })
   }
+
+  public Confirmations(type?: string) {
```

Comment:
would it be more helpful if we narrow down the `string` type here?
maybe: `type?: email_change | password_change`

I'd say it's an OK trade-off since adding a new type and releasing a new version of this library would take 5-mins at most 🤔

Reply:
and we should add types for the return value as well, it's very helpful. maybe you can check `Events` module 👍